drivers/gles2/rasterizer_gles2.cpp

```diff
@@ -226,6 +226,21 @@ void RasterizerGLES2::initialize() {
 			print_line("OpenGL debugging not supported!");
 		}
 	}
+#ifdef DEBUG_ENABLED
+	if (GLAD_GL_KHR_debug) {
+		push_label_func = glPushDebugGroup;
+		pop_label_func = glPopDebugGroup;
+		khr_debug_enabled = true;
+	}
+	if (GLAD_GL_ARB_timer_query) {
+		gen_queries_func = glGenQueries;
+		del_queries_func = glGenQueries;
+		query_counter_func = glQueryCounter;
+		get_query_func = glGetQueryObjectui64v;
+		enable_frame_timings();
+	}
+#endif
+#else
```

Comment:
```suggestion
```

Reply:
Poke @Ansraer

Reply:
On it. Will push an update in a day or two.

Reply:
Still pending :)